<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logWindow</class>
 <widget class="QMainWindow" name="logWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>527</width>
        <height>347</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>527</width>
        <height>347</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_login">
       <widget class="QLabel" name="label_login">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>60</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>221</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_User">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Passward">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="line_L_User"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_L_Pasw">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="bt_Login">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>230</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_Register">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>60</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_nickname">
       <widget class="QLabel" name="labe_nickname1">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>刚注册吧？</string>
        </property>
       </widget>
       <widget class="QLabel" name="labe_nickname2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>85</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>请输入昵称</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_nickname">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>150</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>230</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>决定</string>
        </property>
       </widget>
       <widget class="QPushButton" name="labe_back">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_register">
       <widget class="QLabel" name="label_Register">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>110</y>
          <width>281</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="lable_R_User">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_R_pas">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_R_Rpas">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>再次输入密码</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="line_R_User"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_R_Pasw">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_R_RPasw">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="bt_NowRegister">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>立即注册</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_R_giveup">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_show">
       <widget class="QLabel" name="label_shwoSucceed">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>注册成功</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_welcome">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>欢迎您</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nickname">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>130</y>
          <width>131</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>nick name</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
